aiodogstatsd/client.py

```diff
@@ -219,6 +221,22 @@ def _report(
             )
         )
 
+    @contextmanager
+    def timer(
```

Comment:
1. Let's name it `timeit`
1. `sample_rate` must be `Optional[typedefs.MSampleRate]` as you already did it in the previous PR
1. And better to rewrite a bit in a way:
```
loop = get_event_loop()
started_at = loop.time()
yield
value = (loop.time() - started_at) * 1000
```

Reply:
What is the benefit of using loop.time?

Reply:
It's a recommended way to do such things, you can find the same implementation in many libraries or frameworks (e.g. `aiohttp`).

Reply:
I assumed so, but wondered why exactly (to support different timings of event loops?..), oh well :)